spec/lib/net/ntlm_spec.rb

```diff
@@ -59,6 +59,14 @@
     end
   end
 
+  context 'when the username contains non-ASCI characters' do
+    let(:user) { 'юзер' }
+
+    it 'should return the correct ntlmv2 hash' do
+      expect(Net::NTLM::ntlmv2_hash(user, passwd, domain)).to eq(["ba3d357a20233dfc432b727537272bab"].pack("H*"))
```

Comment:
is this test supposed to fail without this change?

Reply:
I think you need to pass `{unicode: true}` to hit the new codepath you added. How did you calculate the expected value - can you double check this is correct?

Reply:
You're absolutely right, I'll add the `{unicode: true}` flag.

Regarding the hash, I haven't calculated it, but just grabbed the output of `#ntlmv2_hash` to make sure the hash won't change with any future updates. At that time I wrote this specs, I validated this fix against a real Windows and made sure it worked as expected. That being said, I'm fine with removing this RSpec example if you think it doesn't make sense.

Note that I had to update this hash since now it is using the correct `unicode` flag.